src/gpuarray_buffer_opencl.c

```diff
@@ -110,21 +123,24 @@ cl_ctx *cl_make_ctx(cl_context ctx, int flags) {
   const char *rlk[1];
   gpukernel *m;
 
+  e = setup_lib();
+  if (e != GA_NO_ERROR)
+    return NULL;
   id = get_dev(ctx, NULL);
   if (id == NULL) return NULL;
   err = clGetDeviceInfo(id, CL_DEVICE_QUEUE_PROPERTIES, sizeof(qprop),
-                        &qprop, NULL);
+                         &qprop, NULL);
```

Comment:
A first version of this code added a 'g' prefix to all the functions, thus the extra spaces.  I failed to remove all of them.